androidartifact/file_name_test.go

```diff
@@ -44,6 +44,7 @@ func TestParseArtifactPath(t *testing.T) {
 		},
 	}
 	for _, tt := range tests {
+		tt := tt
```

Comment:
I noticed you are rebinding the loop variable to a local variable to avoid the famous Go for loop variable issue. You only need to do this when you are running your tests in parallel but we are not doing that. In our case the sub tests are not running in separate goroutines. You only need to do what you did here when you have this in your tests:
```
func TestParseArtifactPath(t *testing.T) {
    t.Parallel()
    ...
    for _, tt := range tests {
        tt := tt
        t.Run(tt.name, func(t *testing.T) {
             t.Parallel()
             ...
        }
    }
}
```